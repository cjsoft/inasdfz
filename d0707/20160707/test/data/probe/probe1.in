10
15
L 0
L 352
L 0
L 0
E 0
L 0
L 0
E 1257
L 321
E 0
L 0
L 0
E 692
L 321
L 0
15
L 691
E 1119
E 0
L 0
L 1200
L 1119
E 0
E 0
E 1366
E 0
E 0
E 933
L 0
E 430
L 658
15
L 1746
E 0
E 1380
E 1952
L 736
L 493
E 1510
E 829
L 1915
E 0
L 493
L 1746
L 472
E 0
E 795
15
L 1830
L 0
L 611
L 0
E 21
L 1196
E 1778
L 1196
L 611
E 1778
L 973
E 1306
E 647
L 1306
L 407
15
E 1291
E 900
E 874
E 253
E 1397
E 1570
E 595
L 0
L 0
L 0
L 0
L 0
L 0
L 0
L 0
15
E 184
E 91
E 433
E 165
E 1142
E 146
E 1772
L 0
L 0
L 0
L 0
L 0
L 0
L 0
L 0
15
E 0
E 0
E 0
E 0
E 0
E 0
E 0
L 600
L 242
L 1549
L 644
L 146
L 1825
L 1225
L 443
15
E 0
E 0
E 0
E 0
E 0
E 0
E 0
L 660
L 1112
L 1731
L 1064
L 884
L 963
L 22
L 1607
15
L 0
L 0
L 0
L 0
L 0
L 0
L 0
E 0
E 0
E 0
E 0
E 0
E 0
E 0
E 0
15
L 0
L 0
L 0
L 0
L 0
L 0
L 0
L 0
L 0
L 0
L 0
L 0
L 0
L 0
L 0
