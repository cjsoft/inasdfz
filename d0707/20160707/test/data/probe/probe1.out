0
5
6
OTHER
0
0
0
0
8
0
